ssh/tailssh/incubator.go

```diff
@@ -204,6 +204,12 @@ func parseIncubatorArgs(args []string) (a incubatorArgs) {
 // OS, sets its UID and groups to the specified `--uid`, `--gid` and
 // `--groups` and then launches the requested `--cmd`.
 func beIncubator(args []string) error {
+	// To defend against issues like https://golang.org/issue/1435,
```

Comment:
Maybe say how this shouldn't matter on Linux but might help other platforms?

Reply:
@bradfitz - added a bit more comment; PTAL?